### ISSUE TITLE
Setting/Engine crashes

### ISSUE THREAD

**Issue by kennychenfs** [featurecat/lizzie#659]
openjdk version "1.8.0_222"
OpenJDK Runtime Environment (build 1.8.0_222-8u222-b10-1ubuntu1~18.04.1-b10)
OpenJDK 64-Bit Server VM (build 25.222-b10, mixed mode)
Every small window in Settings crashes.

**Comment by toomasr** [comment]
Is it similar to #661 or #662? If yes then those patches should fix it for you too.

**Comment by kennychenfs** [comment]
No, the windows didn't appear.  It looks like a white rectangle.
I think I didn't install some display module, can you provide me some possible ones?

**Comment by toomasr** [comment]
Can you explain again what crashes for you after doing what? We are both on Ubuntu so I can check.

**Comment by kennychenfs** [comment]
I click Settings/Engine, and a white rectangle appears.  Then I can't do anything, nor the main window.  So I must stop the program.

**Comment by kaorahi** [comment]
#642 ?

**Comment by kennychenfs** [comment]
https://github.com/featurecat/lizzie/issues/665
the same situation, what can I do?

kaorahi <notifications@github.com> 於 2019年11月25日 週一 20:12 寫道：

**Comment by toomasr** [comment]
Do you mind compiling from https://github.com/toomasr/lizzie and see if the issue is still there?
